src/cl-change-case.lisp

```diff
@@ -30,7 +28,7 @@
 
 (in-package :cl-change-case)
 
-(define-constant +empty-string+ "" :test #'string=)
+(defvar +empty-string+ "")
```

Comment:
Looks reasonable. Though, expressing that this is an actual constant would be preferable.